src/navigation/breadcrumbs/breadcrumbs.jsx

```diff
@@ -27,21 +28,24 @@ const propTypes = {
      */
     showOnlyPreviousRoute: PropTypes.bool,
     /**
-     * Applied for all the breadcrumbs but the last item
+     * Separation String between breadcrumbs
      */
-    separatorIconType: PropTypes.string,
+    separatorString: PropTypes.string,
```

Comment:
Let's call this divider, sense this is the word we use for this type of thing in all our other components.